composer/metrics/nlp.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, vocab_size: Optional[int] = None, dist_sync_on_step: bool = F
         if vocab_size is not None:
             warnings.warn(
                 DeprecationWarning(
-                    'The vocab_size argument is deprecated and will be removed in 0.14. It is no longer needed, because the correct shape of output and target is inferred based on the number of target elements.'
+                    'The vocab_size argument is deprecated and will be removed in 0.15. It is no longer needed, because the correct shape of output and target is inferred based on the number of target elements.'
```

Comment:
Why are we pushing this deprecation warnings out to 0.15?

Reply:
It is a version too early and doesn't give a 2 release period as we require.